src/base/graphql_printer.re

```diff
@@ -79,7 +79,8 @@ let is_internal_directive = d =>
   | "bsVariant"
   | "bsRecord"
```

Comment:
Would be great to deprecate `bsRecord` as `ppxAs` is recommended from now on. Also other `bs` can be deprecated with note about using the ones with `ppx` prefix. Probably can be implemented as validation step.

Reply:
bsRecord is still relevant when we support compiling to objects, but eventually it can be deprecated indeed.